webrtc-c/canary/jobs/orchestrator.groovy

```diff
@@ -133,10 +133,40 @@ pipeline {
                                 string(name: 'DURATION_IN_SECONDS', value: LONG_RUNNING_DURATION_IN_SECONDS.toString()),
                                 string(name: 'MASTER_NODE_LABEL', value: "ec2-us-west-2"),
                                 string(name: 'VIEWER_NODE_LABEL', value: "ec2-us-west-2"),
-                                string(name: 'RUNNER_LABEL', value: "LongRunning"),
+                                string(name: 'RUNNER_LABEL', value: "WebrtcLongRunning"),
                             ],
                             wait: false
                         )
+
+                        /* TODO: Signaling canary is still unstable. Uncomment below when the software is stable.
```

Comment:
Do we. need this commented block?

Reply:
Not really. But, I'd rather not waste the resources, and try to stabilize the canary locally first.